### PR TITLE
Updated to use centralised CI and --grow format for changelog

### PR DESCRIPTION
**Pull request by alourie**
Signed-off-by: Alex Lourie <alex.lourie@zepben.com>

### CODE REVIEW

File: .github/workflows/python-lib-release.yml [hunk 1]
```diff
@@ -1,158 +1,11 @@
...
-      - uses: actions/checkout@v2
-
-      - name: Update to next minor version
-        run: |
-          /scripts/update-version.sh --python --release setup.py changelog.md
-        shell: bash
+  run:
+    uses: zepben/.github/.github/workflows/python-lib-release.yml@main
+    with:
+      private: true
```
Comment (kgreav) [inline .github/workflows/python-lib-release.yml:9]: this one is a public repo, so this should be false and licencecheck needs to be true as well.

**Review by kgreav — CHANGES_REQUESTED** [review]

File: .github/workflows/python-lib-release.yml (hunk 1 above)
Comment (alourie) [reply .github/workflows/python-lib-release.yml]: Done

**Review by kgreav — CHANGES_REQUESTED** [review]
sorry missed it in my previous review - can you add python-lts-branch.yml and python-hotfix-branch.yml ?

**Comment by alourie** [comment]
> sorry missed it in my previous review - can you add python-lts-branch.yml and python-hotfix-branch.yml ?
yup

**Review by kgreav — APPROVED** [review]

**Pull request merged**